plugins/out_azure/azure.c

```diff
@@ -319,13 +320,35 @@ static int cb_azure_exit(void *data, struct flb_config *config)
     return 0;
 }
 
+/* Configuration properties map */	
+static struct flb_config_map config_map[] = {	
+    {	
```

Comment:
Your patch occasionally have trailing white spaces. Since it is generally
considered to be a bad practice, you'd be better to trim them before
submitting a PR.